addons/l10n_il/models/l10n_il_ita_branch.py

```diff
@@ -0,0 +1,12 @@
+# -*- coding: utf-8 -*-
+
+from odoo import fields, models
+
+
+class ITABranch(models.Model):
+    _name = 'l10n.il.ita.branch'
+
+    name = fields.Char(string='ITA Branch')
+    code = fields.Char(string='ITA Branch Code')
```

Comment:
1st of all, we can not change the schema in v13 for l10n_il.   I would say, again with a selection, but it is possible we need the name also in Hebrew, which is not reflected here...